entrypoint.sh

```diff
@@ -165,13 +173,13 @@ function lint_files() {
 
 	for type in "${filetypes_to_lint[@]}"; do
 		if [[ $type == "all" ]]; then
-			cmd="${linter_array[name]} $linter_args"
+			cmd="${linter_array[name]} $linter_args $(printf '%q ' "${included[@]}")"
```

Comment:
I've rarely, if ever, used printf in bash. Adding the array as ${included[*]} would work but was buggy in some local testing, as it would error when the array contain file names as `./file` . The `.` would break it. ${included[@]} worked, except hadolint had a problem assigning an array to the string var `cmd`. ChatGPT gave me the syntax above, and it worked in all of my local testing on the monopreso and goat folders.

Reply:
This is a good use case for printf. Hats off to the llm